src/todo.js

```diff
@@ -0,0 +1,10 @@
+const Todo = (title, description, dueDate, priority) => {
+  const id = Date.now().toString();
+  const isCheked = false;
+
+  return {
+    title, description, dueDate, priority, id, isCheked,
+  };
+};
+
+export default Todo;
```

Comment:
What i said above regarding `projects.js` should also apply to this file (todo.js).
Creation, deletion logic should be covered here......so that we have ALL the logic in one place

it's easier to debug

You created these modules and instead brought their code inside `logic.js` and mixed with DOM manipulation inside there

So you need to do some organization and really separate them

Reply:
I ask for help from a TSE since I'm working alone in this project, and I took his approach to the project and todo model, but thanks for the proposal.